recipes/pylat/build.sh

```diff
@@ -0,0 +1,16 @@
+cd ${SRC_DIR}
+cd src
+python -m numpy.f2py -c -m calccomf calcCOM.f90
+python -m numpy.f2py -c -m calcdistances calcdistances.f90
+python -m numpy.f2py -c -m ipcorr ipcorr.f90
+#f2py -c -m src/elradial src/elradial.f90
+#f2py -c -m src/siteradial src/siteradial.f90
+
+ln -s ipcorr.*.so ipcorr.so
+ln -s calcdistances.*.so calcdistances.so
+ln -s calccomf.*.so calccomf.so
```

Comment:
This is a bit brittle but the right fix would be upstream. It would be nice to add an issue upstream asking for a better, and more modern, installation scheme.

Reply:
Thank you very much for the suggestion. I agree, we will raise an issue and work on the installation in the following weeks/months.